README.md

```diff
@@ -34,69 +21,89 @@ This plugin supports the following features:
 
 - Sync reactions on posts.
 
-## Installation
```

Comment:
This block is removed but still being referred in introduction.

Reply:
Thanks, @darkLord19! Fixed via https://github.com/mattermost/mattermost-plugin-msteams/pull/653